src/js/renderers/flash.js

```diff
@@ -432,9 +432,12 @@ if (hasFlash) {
 			return 'application/x-mpegURL';
 		} else if (!HAS_MSE && url.includes('.mpd')) {
 			return 'application/dash+xml';
-		} else {
-			return null;
-		}
+		} else if (url.includes('.mp3')) {
```

Comment:
I see. Just keep in mind that we have a shim for MP4 so we may need to add it as well